src/types/chain.ts

```diff
@@ -77,3 +78,27 @@ export interface IRuntime {
 }
 
 export type TSmoldotChain = Chain | void;
+
+export interface IBlockStoreData {
+  number: number;
+  hash: string;
+  timestamp: number;
+  eventsLength: number;
+  validator: string;
+  extrinsics: IExtrinsicStoreData[];
+  identity: Identity;
+}
+export interface IExtrinsicStoreData {
+  id: string;
+  blockNumber: number;
+  signer: string;
+  timestamp: number;
+  isSuccess: boolean;
+  method: string;
+  section: string;
+}
+
+interface Identity {
+  name: string | undefined;
```

Comment:
hmm some times we use `?` some times we use | undefined

Reply:
done